README.md

```diff
@@ -50,7 +50,7 @@ to turn these back into the format that ember-data expects for
 sideloads. It will restructure arbitrarily deeply nested embeds, and it
 will delete `_links` keys when they match embedded properties. Example:
 
-```
+```javascript
```

Comment:
The line after this uses `#` for a comment, which is not js syntax. If you want to change this the next line should also be changed (along with any other examples)